SECURITY.md

```diff
@@ -1,15 +1,39 @@
+
 ## Supported Versions
 
 This is alpha software, do not run on a production system. Notably, we currently provide no migration path not even "dump state and restart" to move to future versions.
 
+We will have a stable v0.x version before the final v1.0.0 version with the same API as the v1.0 version in order to run last testnets and manual testing on it. We have not yet committed to that version number. wasmd 0.22 will support Cosmos SDK 0.44/0.45 and should be quite close to a final API, minus some minor details.
 
-We will have a stable v0.x version before the final v1.0.0 version with the same API as the v1.0 version in order to run last testnets and manual testing on it. We have not yet committed to that version number. 
 Our v1.0.0 release plans were also delayed by upstream release cycles, and we have continued to refine APIs while we can.
 
-## Reporting a Vulnerability
+## Coordinated Vulnerability Disclosure Policy
 
-Please report any security issues via email to security@confio.gmbh. 
+We ask security researchers to keep vulnerabilities and communications around vulnerability submissions private and confidential until a patch is developed. In addition to this, we ask that you:
 
-You will receive a response from us within 48 hours. If the issue is confirmed, we will release a patch as soon as possible depending on complexity but historically within a few days.
+ - Allow us a reasonable amount of time to correct or address security vulnerabilities.
+ - Avoid exploiting any vulnerabilities that you discover.
+ - Demonstrate good faith by not disrupting or degrading services built on top of cw-plus.
+
+## Vulnerability Disclosure Process
+
+Confio uses the following disclosure process for the various CosmWasm-related repos:
+
+ - Once a security report is received, the core development team works to verify the issue.
+ - Patches are prepared for eligible releases in private repositories.
+ - We notify the community that a security release is coming, to give users time to prepare their systems for the update. Notifications can include Discord messages, tweets, and emails to partners and validators.
+ - 24 hours following this notification, the fixes are applied publicly and new releases are issued.
+ - Once releases are available, we notify the community, again, through the same channels as above.
+ - Once the community is notified, we will pay out any relevant bug bounties to submitters.
+ - One week after the releases go out, we will publish a post with further details on the vulnerability as well as our response to it.
+
+ This process can take some time. Every effort will be made to handle the bug in as timely a manner as possible. However, it's important that we follow the process described above to ensure that disclosures are handled consistently and to keep this codebase and the projects that depend on them secure.
```

Comment:
I'm not a native speaker, but "in as timely a manner as possible" sounds like a broken sentence

Reply:
It's grammatical, but an equivalent sentence might be, "Every effort will be made to handle the bug as quickly and thoroughly as possible."